package.json

```diff
@@ -72,9 +70,7 @@
   ],
   "license": "MIT",
   "main": "shrinkpack",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/JamieMason/shrinkpack.git"
+  "repository": "JamieMason/shrinkpack"
```

Comment:
This line is missing a comma, causing `package.json` to be invalid and CI to fail because of that.